src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -872,19 +907,17 @@ private static bool TryGetEasyRequestFromContext(IntPtr context, out EasyRequest
                 return false;
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null)
+            private void EventSourceTrace<TArg0>(string formatMessage, TArg0 arg0, EasyRequest easy = null, [CallerMemberName] string memberName = null)
```

Comment:
No, I think these add value by filling in the agent.

Maybe these methods could just straight delegate to the overloads in the outer CurlHandler without doing any other work. Then you can put the "why we are using generics" comment in the one place that does the work: the CurlHandler methods.